src/main/java/net/neoforged/neoforge/common/extensions/IPlayerExtension.java

```diff
@@ -81,4 +81,11 @@ default boolean mayFly() {
         // TODO 1.20.5: consider forcing mods to use the attribute
         return self().getAbilities().mayfly || self().getAttributeValue(NeoForgeMod.CREATIVE_FLIGHT) > 0;
     }
+
+    /**
+     * {@return whether this player is a fake player}
```

Comment:
Should this javadoc contain a link to FakePlayer class so people know what it means by fake player